### ISSUE TITLE
Add gist diagrams to release distribution

### ISSUE THREAD

**Issue by sa-bpelakh** [semanticarts/gist#259]
The large diagram is already included in the distribution, but is not accessible from our website. The compact diagram should also be generated, and embedded in our gist web page (and possibly the release notes?)

**Comment by MichaelSullivanArchitect** [comment]
I'm taking a slightly different approach (and could use some assistance). In order to get developers interested in KG's, I plan to publish a set of micro-patterns illustrating how a developer can attach mapped content to to a GIST ontology depending on the type of bucket. e.g. Tag, Collection, Group, Category, Ordinal Collection, etc. Simplifying things to buckets is much easier for developers to get their heads around. For example, I'm using gra.fo to implement the micro-patterns. Here's one example of Taxonomy (which may or may not be 100% correct): 
![Protege Taxonomy Bucket](https://user-images.githubusercontent.com/60393887/82066243-23a0e400-969d-11ea-8b41-88a2d7eadca4.jpg)

**Comment by sa-bpelakh** [comment]
@MichaelSullivanArchitect That's on a much more detailed level than the diagrams we currently provide, ours are intended to show the interdependence between the different gist domain ontologies, and  don't really address instance-level relationships.

**Comment by MichaelSullivanArchitect** [comment]
Oh I wasn't suggesting my approach as a replacement at all. Rather, developers need a toolkit of patterns to study (and use) in order to feel comfortable with KG's. My goal is to break down barriers in order to increase acceptance. At the moment, 100% of developers I've been attempting to evangelize about KGs simply throw their hands in the air and roll their eyes. They are still stuck in an application-centric approach.

**Comment by MichaelSullivanArchitect** [comment]
I guess the question is: **is what I'm proposing something that is of interest to the GIST team?** My goal is simply to get more people using semantic technology, regardless of vendor. And after a year of failed attempts at getting developers interested, I believe adopting GIST to "get the ball rolling" may hold the key.

**Comment by marksem** [comment]
Is this still relevant, given the demodularization of gist?

**Comment by rjyounes** [comment]
No. I think it's been removed from the bundling operation - @sa-bpelakh is that correct?

**Comment by Jamie-SA** [comment]
@MichaelSullivanArchitect not sure how I missed this discussion in May. I agree with your suggestion about providing more examples to guide and simplify usage of gist. There is a different issue, #196 that is more appropriate for your suggestions. I will link in your suggestions there and follow up discussion should happen there.

**Comment by sa-bpelakh** [comment]
@rjyounes the diagram that showed inter-ontology dependencies has been removed as OBE, the more detailed diagram as suggested by @MichaelSullivanArchitect , which is closer to our original E6 files, was never created. Do we need and/or want it?

**Comment by uscholdm** [comment]
We all agree: 
- No more Visio/e6 diagrams (makes collaboration impossible)
- No more import hierarchy diagram (now moot)

What would be useful is diagrams illustrating parts of the model and/or examples using it. This is a whole new separate effort. The diagram above is one example.  I agree that this belongs in a separate issue, but https://github.com/semanticarts/gist/issues/196 does not seem to  be exactly this, not hitting it on the head.

Ideally we would illustrate all our patterns with videos of slow reveal presentations.
